src/health-check.php

```diff
@@ -57,6 +57,10 @@
 require_once( dirname( __FILE__ ) . '/includes/class-health-check-troubleshoot.php' );
 require_once( dirname( __FILE__ ) . '/includes/class-health-check-files-integrity.php' );
 require_once( dirname( __FILE__ ) . '/includes/class-health-check-mail-check.php' );
+require_once( dirname( __FILE__ ) . '/includes/class-health-check-wp-debug.php' );
+
+// Restore wp-config backup on deactivation
+register_deactivation_hook( __FILE__, array( 'Health_Check_WP_Debug', 'deactivation_restore_wp_config_backup' ) );
```

Comment:
The backup file should only be there while enabling/disabling the mode, we shouldn't keep one and overwrite on plugin deactivation (we don't know what changes may have been made to the config file that may now be lost).